From 7698e7885e139f26772bea5d21d5db5b6ea2bcb9 Mon Sep 17 00:00:00 2001
From: Matt Porter <mporter@ti.com>
Date: Tue, 11 Sep 2012 15:32:21 -0400
Subject: [PATCH 2/8] of: Add Adafruit and Sitronix vendor prefixes

This commit adds vendor prefixes for Adafruit Industries
and Sitronix Technologies.

Signed-off-by: Matt Porter <mporter@ti.com>
---
 .../devicetree/bindings/vendor-prefixes.txt        |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Documentation/devicetree/bindings/vendor-prefixes.txt b/Documentation/devicetree/bindings/vendor-prefixes.txt
index db4d3af..1336e91 100644
--- a/Documentation/devicetree/bindings/vendor-prefixes.txt
+++ b/Documentation/devicetree/bindings/vendor-prefixes.txt
@@ -4,6 +4,7 @@ This isn't an exhaustive list, but you should add new prefixes to it before
 using them to avoid name-space collisions.
 
 ad	Avionic Design GmbH
+adafruit	Adafruit Industries
 adi	Analog Devices, Inc.
 amcc	Applied Micro Circuits Corporation (APM, formally AMCC)
 apm	Applied Micro Circuits Corporation (APM)
@@ -44,6 +45,7 @@ schindler	Schindler
 sil	Silicon Image
 simtek
 sirf	SiRF Technology, Inc.
+sitronix Sitronix Technology Corp.
 st	STMicroelectronics
 stericsson	ST-Ericsson
 ti	Texas Instruments
-- 
1.7.7.6

